test_cache_key_flow.py

```
@@ -0,0 +1,146 @@
+#!/usr/bin/env python3
+"""
+Test script to verify the cache key flow is working correctly.
+This script simulates the flow from flight price to seat/service availability.
+"""
+
+import requests
+import json
+import time
```

Ruff (0.12.2)

`time` imported but unused

Remove unused import: `time`

(F401)